source/clear-linux/get-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -1,286 +1,336 @@
+
+#. Move your cursor to this field and press the :kbd:`Enter` key.
+
+#. This will bring you to the ``Choose target device for installation``
+   screen as shown in figure 8. Move your cursor to the desired target and
+   press the :kbd:`Enter` key.
+
+   .. figure:: figures/cai5of6.png
+      :scale: 50 %
+      :alt: Choose target device for installation
+
+      Figure 8: :guilabel:`Choose target device for installation`
+
+   In this example we select the single primary partition from our hard drive.
+
+#. With all the |CL| installer setup information gathered for the automatic
+   installation option, the |CL| Installer will prompt you to begin the actual
+   installation as shown in figure 9.
+
+   .. figure:: figures/cai6of6.png
+      :scale: 50 %
+      :alt: Begin installation
+
+      Figure 9: :guilabel:`Begin installation`
+
+   If you are satisfied with the information you have entered, select the
+   :guilabel:`< Yes >` field and press :kbd:`Enter` to begin installing |CL|.
+
+|CL| Installation begins...each step will show it's status as it progresses
+through the automated installation process.
+
+Complete installation process
+=============================
+
+#. Once all steps have completed, you will see the :guilabel:`< Ok >` prompt
+   as shown in figure 10. Press the
+   :kbd:`Enter` key to continue.
+
+   .. figure:: figures/caisuccess.png
+      :scale: 50 %
+      :alt: Installation complete
+
+      Figure 10: :guilabel:`Installation complete`
+
+#. The final screen is shown in figure 11 and you will be prompted that the
+   installation was successful and the system will be rebooted. Press the
+   :kbd:`Enter` key and remove the USB media while the system restarts.
+
+   .. figure:: figures/cairestart.png
+      :scale: 50 %
+      :alt: Successful installation
+
+      Figure 11: :guilabel:`Successful Installation`
+
+Set up your root and user accounts
+==================================
+
+With the USB device removed and the system restarted, your newly installed
+|CL| system boots and presents a full screen console requesting ``login:`` as
+shown in figure 12:
+
+.. figure:: figures/cairoot.png
+   :scale: 50 %
+   :alt: Login screen
+
+   Figure 12: :guilabel:`Login screen`
+
+#. At the initial login prompt, enter: ``root``
+
+#. Once you are prompted, enter a new password
+
+#. Re-enter the password to verify it.
+
+   You have now set your root password and are logged in with root privileges.
+
+**Congratulations!**
+
+You have successfully installed |CL| on a bare metal system using the
+automatic installation method and set the password for the ``root`` user.
+
+The automatic installation of |CL| is designed to install with minimal
+software overhead. Therefore, some housekeeping and package installations
+must occur before you have a full-featured |CL| operating system. These
+instructions are captured in the :ref:`enable-user-space` section:
+
+* Update the OS to its most current version using `swupd`.

```

Comment:
Either include links to every one of these sections or remove the list entirely. You have already cross-referenced the other document. Another alternative would be to display the table of contents of that document.